docs/development_suite/set-up-infrastructure/configure-provider.mdx

```diff
@@ -1,258 +1,150 @@
-- POST - `/api/backend/tenants/{tenantId}/providers`: used to create a new provider;
-- PATCH - `/api/backend/tenants/{tenantId}/providers/{providerId}`: used to edit an existing provider.
+## Providers Overview
 
-:::note
-The `tenantId` refers to the identifier of the company you need to associate the provider with.
-The `providerId` refers to the identifier of the provider created earlier.
-:::
+Mia-Platform Console allows authorized users to manage providers. More specifically, a user can:
 
-:::info
-These APIs are protected and can be used only if you belong to the group `access_token_manager`.
-:::
+* Add a new connection to a provider;
+* Edit the connection to a provider.
+
+To access these features, head to the Company Overview area of the Console by clicking the related button on the home page, next to your company name.
+
+![Go to Company Overview](img/go-to-company-overview.png)
+
+By clicking on "Providers" you will be redirected to the related section, in which (assuming you already have some providers connected) you can see a table containing some information about the providers connected to the Console:
+
+* **Name**: provider name, assigned by the user while connecting the provider to the Console;
+* **Capabilities**: provider capabilities (Git Provider, Secret Manager, CI/CD Tool);
+* **Service**: external reference service on which the provider relies to provide its capabilities (e.g. Azure, GitHub, Jenkins).
+
+Through the search bar at the top of the table you will be able to filter data based on provider name or service, while the bar next to it allows you to filter by one or more capabilities.
+
+![Providers overview](img/providers-overview.png)
+
+## Connect a Provider
+
+You can connect a provider by clicking on the "Add provider connection" button, located in the page's top right corner. A modal will show up, and, through a few steps, you will be asked to enter some information related to the provider.
+
+### Step 1: Provider Services
+
+![Add provider 1a](img/add-provider-1a.png)
+![Add provider 1b](img/add-provider-1b.png)
+
+In this step, you are required to choose the specific provider you want to use among the ones supported by the Console and, for the chosen provider, the capabilities you want to use it for ("Git Provider", "Secret Manager", "CI/CD Tool").
+
+### Step 2: Provider Details
+
+![Add provider 2](img/add-provider-2.png)
+
+In this step, you can insert some general details about your provider:
+
+* **Provider name** (*required*): a human-readable name for the provider;
+* **API Base URL** (*required*): the reference endpoint URL for calls to the provider APIs;
+* **Base URL** (*required*): the reference endpoint URL for the provider;
+* **Description**: a brief description of the provider.
+
+### Step 3: Credentials
+
+![Add provider 3](img/add-provider-3.png)
+
+In this step, you are required to insert information about the authentication method to the provider:
```

Comment:
Qua tipo metterei: "... inserire il metodo di autenticazione che la console dovrà utilizzare per il corretto funzionamento con il provider" od una cosa del genere. Ovvero specificare che serve alla console per avere i "poteri" per fare quello che gli serve

Reply:
Ho aggiornato questa descrizione a riga 68. Per quanto riguarda il banner, esiste già ed è poco più sotto (secondo info box del paragrafo). Gli ho dato un label che ne rende più chiara l'utilità ("Assign the right scope to your credentials")